scm-plugins/scm-git-plugin/src/main/java/sonia/scm/repository/spi/GitMergeStrategy.java

```diff
@@ -107,7 +107,7 @@ ObjectId getRevisionToMerge() {
 
   private String determineMessageTemplate() {
```

Comment:
This does not determine a message template. After the change it determines the message.